app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name: str = name
+        self.appetite: int = appetite
+        self.is_hungry: bool = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
```

Comment:
```suggestion
            return self.appetite
        return 0
```
`else` redundant in this case